doc/howto/many-executables.rst

```diff
@@ -0,0 +1,150 @@
+How to Organize a Project of Many Executables
+=============================================
+
+Suppose that you're following a tutorial that has these chapters, each with many separate OCaml programs:
+
+- Introducing OCaml
+- Introducing Lwt
+- Introducing Hardcaml
```

Comment:
I don't know, this seems like somewhat of an odd premise. One would expect these tutorials to exist but they don't really so honestly I would just drop it and talk in general about how to have multiple executables in the project.

Reply:
How I personally followed books like this is with hundreds of numbered files in a single directory without further organization, which can still be done more neatly than might be obvious with dune -- with a `(names ...)` stanza combining many executables, or with `dune init exe`, vs. what I've done which is to edit dune every time to add a new stanza by hand.

This doc's layout neatly fits something like https://pleac.sourceforge.net/pleac_ocaml/

A recent book, though with only one chapter of example programs, is https://usr.lmf.cnrs.fr/lpo/

A more generally useful doc is probably something like "dune workflow vignettes". Or recipes, following the O'Reilly Cookbook format.